adapters/bidstack/params_test.go

```diff
@@ -9,6 +9,9 @@ import (
 
 var validParams = []string{
 	`{"publisherId": "355cf800-8348-433a-9d95-70345fa70afc"}`,
+	`{"publisherId": "355cf800-8348-433a-9d95-70345fa70afc","placementId":"Some placement ID"}`,
+	`{"publisherId": "355cf800-8348-433a-9d95-70345fa70afc","consent":false}`,
+	`{"publisherId": "355cf800-8348-433a-9d95-70345fa70afc","placementId":"Some placement ID","consent":true}`,
 }
 
 var invalidParams = []string{
```

Comment:
Nitpick: I suggest adding `{"consent":"true"}` to `invalidParams` if the string representation is not meant to be supported.